src/views/Notifications/Home/_components/_utils.tsx

```diff
@@ -104,6 +104,44 @@ export const ActivityCTA = ({
   return <></>;
 };
 
+/**
+ * In the Daily Digest Notification center, we want to show only the first five changes and hide everything else under a `+X more changes`.
+ *
+ * In order to determine what actually changed, this function does the following:
+ * * combs through the `changes` object
+ * * filters out any values that are empty string, null, and empty arrays, indicating there is no change
+ * * puts remaining values into an array.
+ *
+ * Further splicing of the array will then only show the first five changes and the rest will be under the `+X more changes`
+ *
+ * A few exceptions:
+ * * Model Lead changes are important and prepending it to the front of the array ensures that a model lead addition will always show up, rather than being spliced out.
+ *     * Model Lead being added to the Model Plan. The object looks like the following:
+ *     ```
+ *        "modelLeads": {
+            "added": [
+                {
+                    ...
+                    "commonName": "MINT Doe"
+                }
+            ]
+          },
+        ```
+ * * `crTdls` and `planDisucssions`'s object are identical. So rather than having its object go through the filter, this function will push the key if it is 'crTdls' & 'planDiscussions'
+
+      ```
+      "crTdls": {
+          "activity": true
+      },
+      "planDiscussions": {
+          "activity": true
+      }
+      ```
+ *
+ *
+ * @param obj `changes` object passed in from analyzedAudits.
+ *
+ */
```

Comment:
Wow!  This is so helpful at understanding what this logic is doing!  Thanks for this!!!